client/src/components/SearchAppointment.vue

```diff
@@ -39,7 +41,7 @@
           <td> {{item['room']}}</td>
           <td> {{item['booking_type']}}</td>
           <td>
-            <button :id="item['id']"> Book Appointment</button>
+            <button :id="item['id']" v-on:click="bookAppointment($event)"> Book Appointment</button>
```

Comment:
should be "Add to cart" or some variation, because we are not doing the actual booking at this point